pages/api/auth/[...nextauth].js

```diff
@@ -0,0 +1,19 @@
+import NextAuth from 'next-auth'
+import EmailProvider from 'next-auth/providers/email'
+
+export default NextAuth({
+  // Configure one or more authentication providers
+  providers: [
+    EmailProvider({
+      server: {
+        host: process.env.EMAIL_SERVER_HOST,
+        port: +process.env.EMAIL_SERVER_PORT,
```

Comment:
This `+` is a typo?

Reply:
Ah no, is casting to a Number, if you think is confusing I can do a more explicit casting